src/components/JsonWebMap.vue

```diff
@@ -59,6 +60,22 @@ watch(
   },
   { immediate: true }
 )
+
+function getParent(id: string): SelectableItem | undefined {
+  return (parameters.value.selectableItems ?? [])
+    .filter((item: SelectableItem) => {
```

Comment:
It's a detail but I think using "find" function makes more sense : 
`selectableItems.find((item: SelectableItem) => item.children && item.children.find( (child: SelectableSingleItem) => child.id == id) !== undefined)`